lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -60,6 +60,7 @@ public class PrimitiveColumnWriter
     private static final int INSTANCE_SIZE = instanceSize(PrimitiveColumnWriter.class);
     private static final int MINIMUM_OUTPUT_BUFFER_CHUNK_SIZE = 8 * 1024;
     private static final int MAXIMUM_OUTPUT_BUFFER_CHUNK_SIZE = 2 * 1024 * 1024;
+    private static final int MAX_BINARY_STATISTICS_LENGTH = 1024;
```

Comment:
it's used for binary and for varchars too

`MAX_STATISTICS_LENGTH_IN_BYTES`

Reply:
Binary primitive in parquet includes varchars, but I'll update the name